CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Azure Kubernetes Service Changelog
 
+## Release Notes 2025-10-12
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* AKS is blocking creation of new clusters using Availability Sets (VMAS) effective September 30, 2025. Existing VMAS clusters must migrate to Virtual Machine Scale Sets (VMSS) before the retirement date. For more information, see [Availability Sets retirement](https://learn.microsoft.com/azure/aks/availability-sets-on-aks#availability-sets-retirement-date).
+* AKS is now blocking creation of new clusters with Basic Load Balancer following the complete retirement announcement. Affected clusters must migrate to Standard Load Balancer. For more information, see [Basic Load Balancer retirement announcement](https://learn.microsoft.com/en-us/answers/questions/1033471/retirement-announcement-basic-load-balancer-will-b).
```

Comment:
- on first line in place of 'following the complete retirement announcement',  I suggest 'retired on 30 September 2025' to be explicit on when the retirement happened.